application/controllers/UploaderController.php

```diff
@@ -274,7 +272,7 @@ public function run($actionID)
                     $return = array(
                         "success" => true,
                         "size"    => $size,
-                        "name"    => rawurlencode(basename($filename)),
+                        "name"    => basename($filename),
```

Comment:
basename still needed ? Since it's sanitised before ?

Reply:
I also think it can be removed, especially considering the  locale problems (which I unfortunately could not reproduce in Windows).
Will do.